patches/minecraft/net/minecraft/client/gui/screens/worldselection/WorldSelectionList.java.patch

```diff
@@ -0,0 +1,34 @@
+--- a/net/minecraft/client/gui/screens/worldselection/WorldSelectionList.java
++++ b/net/minecraft/client/gui/screens/worldselection/WorldSelectionList.java
+@@ -66,6 +_,7 @@
+    static final DateFormat f_101646_ = new SimpleDateFormat();
+    static final ResourceLocation f_101647_ = new ResourceLocation("textures/misc/unknown_server.png");
+    static final ResourceLocation f_101648_ = new ResourceLocation("textures/gui/world_selection.png");
++   private static final ResourceLocation FORGE_EXPERIMENTAL_WARNING_ICON = new ResourceLocation("forge","textures/gui/experimental_warning.png");
+    static final Component f_101649_ = (new TranslatableComponent("selectWorld.tooltip.fromNewerVersion1")).m_130940_(ChatFormatting.RED);
+    static final Component f_101650_ = (new TranslatableComponent("selectWorld.tooltip.fromNewerVersion2")).m_130940_(ChatFormatting.RED);
+    static final Component f_101651_ = (new TranslatableComponent("selectWorld.tooltip.snapshot1")).m_130940_(ChatFormatting.GOLD);
+@@ -209,6 +_,7 @@
+          RenderSystem.m_69478_();
+          GuiComponent.m_93133_(p_101721_, p_101724_, p_101723_, 0.0F, 0.0F, 32, 32, 32, 32);
+          RenderSystem.m_69461_();
++         renderExperimentalWarning(p_101721_, p_101727_, p_101728_, p_101723_, p_101724_);
+          if (this.f_101693_.f_91066_.f_92051_ || p_101729_) {
+             RenderSystem.m_157456_(0, WorldSelectionList.f_101648_);
+             GuiComponent.m_93172_(p_101721_, p_101724_, p_101723_, p_101724_ + 32, p_101723_ + 32, -1601138544);
+@@ -529,6 +_,15 @@
+ 
+       public String m_170324_() {
+          return this.f_101695_.m_78361_();
++      }
++      private void renderExperimentalWarning(PoseStack stack, int mouseX, int mouseY, int top, int left) {
++         if(f_101695_.isExperimental()) {
++            RenderSystem.m_157456_(0, WorldSelectionList.FORGE_EXPERIMENTAL_WARNING_ICON);
++            GuiComponent.m_93133_(stack,left +  WorldSelectionList.this.m_5759_() - 36, top, 0.0F, 0.0F, 32, 32, 32, 32);
++            RenderSystem.m_157456_(0, this.f_101698_ != null ? this.f_101696_ : WorldSelectionList.f_101647_);
```

Comment:
This line should have a comment explaining what it does -- resetting the shader texture to what it was before rendering the Forge experimental warning icon.